internal/services/api_token/schema.go

```diff
@@ -101,6 +110,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 						"expired",
 					),
 				},
+				Default: stringdefault.StaticString("active"),
```

Comment:
This can be codegen'd for you as well:

```diff
      status:
        description: Status of the token.
        type: string
        enum:
          - active
          - disabled
          - expired
        example: active
        x-auditable: true
+       default: active
```